daprdocs/content/en/operations/monitoring/jaeger.md

```diff
@@ -0,0 +1,106 @@
+---
+type: docs
+title: "How-To: Set up Jaeger for distributed tracing"
+linkTitle: "Jaeger"
+weight: 3000
+description: "Set up Jaeger for distributed tracing"
+type: docs
+---
+
+The following steps shows you how to configure Dapr to send distributed tracing data to Jaeger running as a container in your Kubernetes cluster, how to view them.
+
+## Setup
+
+First the following YAML file to install Jaeger
+* jaeger-operator.yaml
+```yaml
+apiVersion: jaegertracing.io/v1
+kind: "Jaeger"
+metadata:
+  name: "jaeger"
+spec:
+  strategy: allInOne
+  ingress:
+    enabled: false
+  allInOne:
+    image: jaegertracing/all-in-one:1.13
+    options:
+      query:
+        base-path: /jaeger
+```
+
+Now, let's use that Jaeger Operator resource to install Jaeger
+```bash
+# Install Jaeger
+helm repo add jaegertracing https://jaegertracing.github.io/helm-charts
+helm install jaeger-operator jaegertracing/jaeger-operator
+kubectl apply -f jaeger-operator.yaml
+
+# Wait for Jaeger to be up and running
+kubectl wait deploy --selector app.kubernetes.io/name=jaeger --for=condition=available
+```
+
+Next, create the following YAML files locally:
+
+* jaeger.yaml component
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: zipkin
+spec:
+  # Note that since we are using the Zipkin protocol to talk to Jaeger, the component
```

Comment:
I'd pull this comment out of the YAML file and explicitly explain this before this part, this will surely be a confusing part of the instruction. Also, elaborate if possible (Dapr currently supports two kinds of exporters, since Jaeger is compatible with Zipkin... and so on...)

Reply:
Done